### ISSUE TITLE
Problems parsing diff blocks which with certain header statements

### ISSUE THREAD

**Issue by ssvendsen** [rtfpessoa/diff2html#95]
It seems the commit tagged as v2.0.0-rc.9 introduced some problems with parsing certain block header statements.

In this commit, lines 2575 in diff2html.js changed from 

```
      if (currentFile && utils.startsWith(line, hunkHeaderPrefix)) {
        startBlock(line);
        return;
      }
```

to

```
      if (
        (currentFile && utils.startsWith(line, hunkHeaderPrefix)) ||
        (currentFile.isGitDiff && currentFile && currentFile.oldName && currentFile.newName && !currentBlock)
      ) {
        startBlock(line);
        return;
      }
```

A block is now potentially started _before_ the hunkHeaderPrefix when parsing a diff produced by git with the --find-renames flag specified, leading to garbage in the displayed html diff. 

An example of a block header as produced by git diff --find-renames:

```
diff --git a/****** b/******
similarity index 98%
rename from C:/******
rename to C:/******
index e01513b..f14a870 100644
--- a/C:/******
+++ b/C:/******
@@ -1,4 +1,32 @@
```

**Comment by rtfpessoa** [comment]
@ssvendsen I need to take a look. I have a really hard time parsing both Git Diff and Unified diff.

If you have any code suggestion it would be really appreciated.

Even though I will try fix this.

**Comment by rtfpessoa** [comment]
@ssvendsen this should be fixed in version 2.0.3

**Comment by ssvendsen** [comment]
Thanks, I'll give it  try!
